pkgs/by-name/zm/zmkBATx/package.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, qt6
+, pkg-config
+, dbus
+, simpleBluez
+}:
+let
+  version = "1.0.1";
+  pname = "zmkBATx";
+  src = fetchFromGitHub {
+    owner = "mh4x0f";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-xbiwRHVTuaZDH3RZlMK2CpKBThtS8g6q5r3C+OccDZg=";
+  };
+in
+stdenv.mkDerivation {
+  inherit pname version src;
+
+  nativeBuildInputs = [
+    qt6.wrapQtAppsHook
+    pkg-config
+  ];
+
+  buildInputs = [
+    qt6.qtbase
+    qt6.qtconnectivity
+    dbus.lib
+    simpleBluez
+  ];
+
+  patchPhase = ''
+    runHook prePatch
+    substituteInPlace zmkBATx.pro --replace "/usr/include/dbus-1.0" "${dbus.dev}/include/dbus-1.0"
+    substituteInPlace zmkBATx.pro --replace "/usr/lib/x86_64-linux-gnu/dbus-1.0/include" "${dbus.lib}/lib/dbus-1.0/include"
+    substituteInPlace zmkBATx.pro --replace "PREFIX = /usr/local" "PREFIX = $out"
+    runHook postPatch
+  '';
+
+  configurePhase = ''
+    runHook preConfigure
+    qmake zmkBATx.pro LIBS+="-L$src/thirdparty"
```

Comment:
Is there a reason to use a custom configurePhase and not attempt to devendor thirdparty libs (one of which is already included as a new package on this PR?)

I'll attempt to build this later and investigate then (I'm on mobile), but I figured I'd go ahead and ask

Reply:
No, I'm just noob, just have done it.